### DISCUSSION TITLE
Możliwość podania checkout-form-uuid zamiast ręcznie danych adresowych do wygenerowania przesyłki w WzA

### DISCUSSION THREAD

**Discussion by q3ok** [allegro/allegro-api#9217] [Suggestions - resolved]
Pochodne od https://github.com/allegro/allegro-api/issues/9214

Aktualnie nowy zasób shipment-management wymaga rozdzielenia adresu na nazwę ulicy i numer budynku, co jest podawane ciągiem w checkout-forms.

Proponuję, żeby zamiast podawać dane adresowe ręcznie przy generowaniu nowej przesyłki, móc podać tylko uuid checkout-form, a w przypadku potrzeby nadpisania adresu z zamówienia - umożliwić podanie adresu docelowego.

Czyli reasumując:
POST /shipment-management/shipments/create-commands
segment 'receiver' LUB uuid checkout-form

**Comment by MaciejFrackowiak** [comment]
Zaproponowaliśmy [nowe podejście](https://developer.allegro.pl/news/wysylam-z-allegro-optymalizujemy-proces-nadania-przesylki-od-dzisiaj-dla-kazdego-zamowienia-pobierzesz-proponowane-dane-i-ustawienia-dostawy-potrzebne-do-nadania-przesylki-zlOzlovz3Ia) bazujące na danych pobieranych bezpośrednio z zamówienia wraz z dostępnymi opcjami dostawy.
